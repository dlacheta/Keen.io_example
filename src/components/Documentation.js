import React from 'react';

const Documentation = () => (
  <React.Fragment>
    <h2>Documentation</h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed fringilla diam. Donec erat
      mauris, commodo at sodales quis, pretium ac ligula. Donec aliquet eget ipsum vitae maximus.
      Nam lobortis dignissim elit, ut ultricies elit molestie ac. Donec varius egestas sagittis.
      Duis eget dui viverra, ultrices urna vitae, commodo justo. Vivamus varius dolor vel urna
      pellentesque, at finibus metus sollicitudin. Nunc in erat metus. Praesent ut ornare mi.
      Quisque congue libero a ipsum sodales molestie. Vestibulum lobortis libero a dignissim cursus.
      Suspendisse mi augue, lacinia sit amet purus ut, eleifend vehicula elit. Maecenas in sem nec
      urna scelerisque porttitor. Quisque pulvinar enim ut dapibus laoreet. Sed eleifend, leo vitae
      cursus cursus, ex nibh sagittis velit, id laoreet libero enim vitae neque.
    </p>
    <p>
      Aliquam convallis enim vel arcu tincidunt, id aliquam enim convallis. Vestibulum ante ipsum
      primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis accumsan sollicitudin
      scelerisque. Phasellus quis lobortis diam. Pellentesque bibendum, dui vel pellentesque
      malesuada, lorem lorem malesuada neque, a fringilla felis justo in urna. Nullam fringilla ex
      risus, nec accumsan enim vehicula sit amet. In ac sapien finibus est placerat lobortis eu quis
      magna. Cras quam odio, blandit vitae vehicula nec, pellentesque non lorem. Pellentesque
      tincidunt arcu quis velit consequat eleifend. Ut volutpat turpis viverra nulla finibus, non
      maximus dolor laoreet. Nam ipsum nisi, iaculis ut mattis congue, malesuada vel urna. Cras id
      ipsum porta, rhoncus diam vel, semper odio. Nunc commodo in lorem in interdum. Nam eu lectus
      condimentum, vehicula mauris eget, aliquam nunc. Sed porttitor justo nec condimentum
      consequat.
    </p>
  </React.Fragment>
);

export default Documentation;
