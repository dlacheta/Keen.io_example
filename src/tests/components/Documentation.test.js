import React from 'react';
import { shallow } from 'enzyme';
import Documentation from '../../components/Documentation';

it('Documentation component renders correctly', () => {
  const wrapper = shallow(<Documentation />);
  expect(wrapper).toMatchSnapshot();
});
