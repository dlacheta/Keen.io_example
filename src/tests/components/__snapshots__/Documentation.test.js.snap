// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Documentation component renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Documentation />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h2>
          Documentation
        </h2>,
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed fringilla diam. Donec erat mauris, commodo at sodales quis, pretium ac ligula. Donec aliquet eget ipsum vitae maximus. Nam lobortis dignissim elit, ut ultricies elit molestie ac. Donec varius egestas sagittis. Duis eget dui viverra, ultrices urna vitae, commodo justo. Vivamus varius dolor vel urna pellentesque, at finibus metus sollicitudin. Nunc in erat metus. Praesent ut ornare mi. Quisque congue libero a ipsum sodales molestie. Vestibulum lobortis libero a dignissim cursus. Suspendisse mi augue, lacinia sit amet purus ut, eleifend vehicula elit. Maecenas in sem nec urna scelerisque porttitor. Quisque pulvinar enim ut dapibus laoreet. Sed eleifend, leo vitae cursus cursus, ex nibh sagittis velit, id laoreet libero enim vitae neque.
        </p>,
        <p>
          Aliquam convallis enim vel arcu tincidunt, id aliquam enim convallis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis accumsan sollicitudin scelerisque. Phasellus quis lobortis diam. Pellentesque bibendum, dui vel pellentesque malesuada, lorem lorem malesuada neque, a fringilla felis justo in urna. Nullam fringilla ex risus, nec accumsan enim vehicula sit amet. In ac sapien finibus est placerat lobortis eu quis magna. Cras quam odio, blandit vitae vehicula nec, pellentesque non lorem. Pellentesque tincidunt arcu quis velit consequat eleifend. Ut volutpat turpis viverra nulla finibus, non maximus dolor laoreet. Nam ipsum nisi, iaculis ut mattis congue, malesuada vel urna. Cras id ipsum porta, rhoncus diam vel, semper odio. Nunc commodo in lorem in interdum. Nam eu lectus condimentum, vehicula mauris eget, aliquam nunc. Sed porttitor justo nec condimentum consequat.
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Documentation",
        },
        "ref": null,
        "rendered": "Documentation",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed fringilla diam. Donec erat mauris, commodo at sodales quis, pretium ac ligula. Donec aliquet eget ipsum vitae maximus. Nam lobortis dignissim elit, ut ultricies elit molestie ac. Donec varius egestas sagittis. Duis eget dui viverra, ultrices urna vitae, commodo justo. Vivamus varius dolor vel urna pellentesque, at finibus metus sollicitudin. Nunc in erat metus. Praesent ut ornare mi. Quisque congue libero a ipsum sodales molestie. Vestibulum lobortis libero a dignissim cursus. Suspendisse mi augue, lacinia sit amet purus ut, eleifend vehicula elit. Maecenas in sem nec urna scelerisque porttitor. Quisque pulvinar enim ut dapibus laoreet. Sed eleifend, leo vitae cursus cursus, ex nibh sagittis velit, id laoreet libero enim vitae neque.",
        },
        "ref": null,
        "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed fringilla diam. Donec erat mauris, commodo at sodales quis, pretium ac ligula. Donec aliquet eget ipsum vitae maximus. Nam lobortis dignissim elit, ut ultricies elit molestie ac. Donec varius egestas sagittis. Duis eget dui viverra, ultrices urna vitae, commodo justo. Vivamus varius dolor vel urna pellentesque, at finibus metus sollicitudin. Nunc in erat metus. Praesent ut ornare mi. Quisque congue libero a ipsum sodales molestie. Vestibulum lobortis libero a dignissim cursus. Suspendisse mi augue, lacinia sit amet purus ut, eleifend vehicula elit. Maecenas in sem nec urna scelerisque porttitor. Quisque pulvinar enim ut dapibus laoreet. Sed eleifend, leo vitae cursus cursus, ex nibh sagittis velit, id laoreet libero enim vitae neque.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Aliquam convallis enim vel arcu tincidunt, id aliquam enim convallis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis accumsan sollicitudin scelerisque. Phasellus quis lobortis diam. Pellentesque bibendum, dui vel pellentesque malesuada, lorem lorem malesuada neque, a fringilla felis justo in urna. Nullam fringilla ex risus, nec accumsan enim vehicula sit amet. In ac sapien finibus est placerat lobortis eu quis magna. Cras quam odio, blandit vitae vehicula nec, pellentesque non lorem. Pellentesque tincidunt arcu quis velit consequat eleifend. Ut volutpat turpis viverra nulla finibus, non maximus dolor laoreet. Nam ipsum nisi, iaculis ut mattis congue, malesuada vel urna. Cras id ipsum porta, rhoncus diam vel, semper odio. Nunc commodo in lorem in interdum. Nam eu lectus condimentum, vehicula mauris eget, aliquam nunc. Sed porttitor justo nec condimentum consequat.",
        },
        "ref": null,
        "rendered": "Aliquam convallis enim vel arcu tincidunt, id aliquam enim convallis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis accumsan sollicitudin scelerisque. Phasellus quis lobortis diam. Pellentesque bibendum, dui vel pellentesque malesuada, lorem lorem malesuada neque, a fringilla felis justo in urna. Nullam fringilla ex risus, nec accumsan enim vehicula sit amet. In ac sapien finibus est placerat lobortis eu quis magna. Cras quam odio, blandit vitae vehicula nec, pellentesque non lorem. Pellentesque tincidunt arcu quis velit consequat eleifend. Ut volutpat turpis viverra nulla finibus, non maximus dolor laoreet. Nam ipsum nisi, iaculis ut mattis congue, malesuada vel urna. Cras id ipsum porta, rhoncus diam vel, semper odio. Nunc commodo in lorem in interdum. Nam eu lectus condimentum, vehicula mauris eget, aliquam nunc. Sed porttitor justo nec condimentum consequat.",
        "type": "p",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h2>
            Documentation
          </h2>,
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed fringilla diam. Donec erat mauris, commodo at sodales quis, pretium ac ligula. Donec aliquet eget ipsum vitae maximus. Nam lobortis dignissim elit, ut ultricies elit molestie ac. Donec varius egestas sagittis. Duis eget dui viverra, ultrices urna vitae, commodo justo. Vivamus varius dolor vel urna pellentesque, at finibus metus sollicitudin. Nunc in erat metus. Praesent ut ornare mi. Quisque congue libero a ipsum sodales molestie. Vestibulum lobortis libero a dignissim cursus. Suspendisse mi augue, lacinia sit amet purus ut, eleifend vehicula elit. Maecenas in sem nec urna scelerisque porttitor. Quisque pulvinar enim ut dapibus laoreet. Sed eleifend, leo vitae cursus cursus, ex nibh sagittis velit, id laoreet libero enim vitae neque.
          </p>,
          <p>
            Aliquam convallis enim vel arcu tincidunt, id aliquam enim convallis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis accumsan sollicitudin scelerisque. Phasellus quis lobortis diam. Pellentesque bibendum, dui vel pellentesque malesuada, lorem lorem malesuada neque, a fringilla felis justo in urna. Nullam fringilla ex risus, nec accumsan enim vehicula sit amet. In ac sapien finibus est placerat lobortis eu quis magna. Cras quam odio, blandit vitae vehicula nec, pellentesque non lorem. Pellentesque tincidunt arcu quis velit consequat eleifend. Ut volutpat turpis viverra nulla finibus, non maximus dolor laoreet. Nam ipsum nisi, iaculis ut mattis congue, malesuada vel urna. Cras id ipsum porta, rhoncus diam vel, semper odio. Nunc commodo in lorem in interdum. Nam eu lectus condimentum, vehicula mauris eget, aliquam nunc. Sed porttitor justo nec condimentum consequat.
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Documentation",
          },
          "ref": null,
          "rendered": "Documentation",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed fringilla diam. Donec erat mauris, commodo at sodales quis, pretium ac ligula. Donec aliquet eget ipsum vitae maximus. Nam lobortis dignissim elit, ut ultricies elit molestie ac. Donec varius egestas sagittis. Duis eget dui viverra, ultrices urna vitae, commodo justo. Vivamus varius dolor vel urna pellentesque, at finibus metus sollicitudin. Nunc in erat metus. Praesent ut ornare mi. Quisque congue libero a ipsum sodales molestie. Vestibulum lobortis libero a dignissim cursus. Suspendisse mi augue, lacinia sit amet purus ut, eleifend vehicula elit. Maecenas in sem nec urna scelerisque porttitor. Quisque pulvinar enim ut dapibus laoreet. Sed eleifend, leo vitae cursus cursus, ex nibh sagittis velit, id laoreet libero enim vitae neque.",
          },
          "ref": null,
          "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed fringilla diam. Donec erat mauris, commodo at sodales quis, pretium ac ligula. Donec aliquet eget ipsum vitae maximus. Nam lobortis dignissim elit, ut ultricies elit molestie ac. Donec varius egestas sagittis. Duis eget dui viverra, ultrices urna vitae, commodo justo. Vivamus varius dolor vel urna pellentesque, at finibus metus sollicitudin. Nunc in erat metus. Praesent ut ornare mi. Quisque congue libero a ipsum sodales molestie. Vestibulum lobortis libero a dignissim cursus. Suspendisse mi augue, lacinia sit amet purus ut, eleifend vehicula elit. Maecenas in sem nec urna scelerisque porttitor. Quisque pulvinar enim ut dapibus laoreet. Sed eleifend, leo vitae cursus cursus, ex nibh sagittis velit, id laoreet libero enim vitae neque.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Aliquam convallis enim vel arcu tincidunt, id aliquam enim convallis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis accumsan sollicitudin scelerisque. Phasellus quis lobortis diam. Pellentesque bibendum, dui vel pellentesque malesuada, lorem lorem malesuada neque, a fringilla felis justo in urna. Nullam fringilla ex risus, nec accumsan enim vehicula sit amet. In ac sapien finibus est placerat lobortis eu quis magna. Cras quam odio, blandit vitae vehicula nec, pellentesque non lorem. Pellentesque tincidunt arcu quis velit consequat eleifend. Ut volutpat turpis viverra nulla finibus, non maximus dolor laoreet. Nam ipsum nisi, iaculis ut mattis congue, malesuada vel urna. Cras id ipsum porta, rhoncus diam vel, semper odio. Nunc commodo in lorem in interdum. Nam eu lectus condimentum, vehicula mauris eget, aliquam nunc. Sed porttitor justo nec condimentum consequat.",
          },
          "ref": null,
          "rendered": "Aliquam convallis enim vel arcu tincidunt, id aliquam enim convallis. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Duis accumsan sollicitudin scelerisque. Phasellus quis lobortis diam. Pellentesque bibendum, dui vel pellentesque malesuada, lorem lorem malesuada neque, a fringilla felis justo in urna. Nullam fringilla ex risus, nec accumsan enim vehicula sit amet. In ac sapien finibus est placerat lobortis eu quis magna. Cras quam odio, blandit vitae vehicula nec, pellentesque non lorem. Pellentesque tincidunt arcu quis velit consequat eleifend. Ut volutpat turpis viverra nulla finibus, non maximus dolor laoreet. Nam ipsum nisi, iaculis ut mattis congue, malesuada vel urna. Cras id ipsum porta, rhoncus diam vel, semper odio. Nunc commodo in lorem in interdum. Nam eu lectus condimentum, vehicula mauris eget, aliquam nunc. Sed porttitor justo nec condimentum consequat.",
          "type": "p",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
